### PR TITLE
HiPACE++: FFTW+OpenMP

### PR DESCRIPTION
**Pull request by ax3l**
- change the default compute backend: we just start to add OpenMP support
- FFTW: prefer with OpenMP acceleration for OpenMP compute backend

X-ref: https://github.com/Hi-PACE/hipace/pull/541

cc @SeverinDiederichs @maxthevenet

### CODE REVIEW

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
@ax3l Feel free to merge this if maintainers are ok with the PR.

**Review by MaxThevenet — APPROVED** [review]
Yes that makes sense to me, thanks!

**Review by SeverinDiederichs — APPROVED** [review]

**Pull request merged**
